src/android/binding.rs

```diff
@@ -216,6 +216,9 @@ fn handle_request(
       let response_headers = {
         let headers_map = JMap::from_env(env, &obj)?;
         for (name, value) in headers.iter() {
+          if name == CONTENT_TYPE || name == CONTENT_LENGTH {
```

Comment:
let's add a comment to explain why we are skipping these

Reply:
It's added now :D